sdk/eventhub/azure-eventhub-checkpointstoretable/azure/eventhub/extensions/checkpointstoretable/_tablestoragecs.py

```diff
@@ -22,17 +50,224 @@ class TableCheckpointStore:
         The hostname of the secondary endpoint.
```

Comment:
I can't comment on the api_version line directly, but it says the default value is '2019-07-07'.

However it looks like the default value is actually '2018-03-28' if we look at the tables package:
https://github.com/Azure/azure-sdk-for-python/blob/d63dcf0b26e6ce6001eb203c7f24594958109644/sdk/tables/azure-data-tables/azure/data/tables/_constants.py#L9

Reply:
fixed it